pkg/config/app/kuma-dp/config_test.go

```diff
@@ -133,19 +133,28 @@ var _ = Describe("Config", func() {
 		err := config.Load(filepath.Join("testdata", "invalid-config.input.yaml"), &cfg)
 
 		// then
-		fmt.Println(err.Error())
+		// fmt.Println(err.Error())
```

Comment:
I think we can just remove this

Reply:
Yeah though it's used for updating the assert. I guess a less lazy me would have converted that to golden files.

Reply:
Converted to golen file